fosspay/inventory.py

```diff
@@ -0,0 +1,62 @@
+"""
+inventory.py
+Stripe Payments Demo. Created by Adrienne Dreyfus (@adrind).
+
+Simple library to store and interact with products and SKUs.
+These methods are using the Stripe Product API, but we tried to abstract them
+from the main code if you'd like to use your own product and order management system instead.
+"""
+
+import stripe
+import os
+from functools import reduce
+from fosspay.stripe_types import Product
+from dotenv import load_dotenv, find_dotenv
+from fosspay.config import _cfg, _cfgi
+
+load_dotenv(find_dotenv())
+
+stripe.api_key = _cfg("stripe-secret")
+stripe.api_version = '2019-03-14'
+
+# For product retrieval and listing set API version to 2018-02-28 so that skus are returned.
+product_api_stripe_version = '2018-02-28'
+
+
+class Inventory:
+    @staticmethod
+    def calculate_payment_amount(items: list) -> int:
+        product_list = stripe.Product.list(
+            limit=1, stripe_version=product_api_stripe_version)
```

Comment:
there are no products created on Stripe, so this `calculate_payment_amount` should be removed